app/views/users/registrations/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "users/shared/error_messages", resource: resource %>
+
+   <div class="field">
```

Comment:
After following the steps I mentioned above I was able to register, confirm email, and login successfully. Great job!!! 👏🏾 

[OPTIONAL] Kindly fix the width and height for the user profile image so that images are adjusted to consistent sizes for all user profiles.

![image](https://github.com/Ezzywealth/rails_blog/assets/38088818/743e4319-13e1-4867-ad82-434cdc8da003)